### PR TITLE
T-4: Setup i18n support

### PR DESCRIPTION
**Pull request by VladyslavGilyuk**
feat: i18 immersed in forms in English and Ukrainian for every text (exept 'Dashboard Kit' (proper name))
fix: all comments removed

Updated!: 
feat: useFormHelpers hook created to include   'const { t } = useTranslation();' for both SingIn/UpFormHelper.
fix: Translations divided into two files 'en/uk.json'
fix: LanguageSwitcher buttons remade with MUI

### CODE REVIEW

File: src/i18n.ts [hunk 1]
```diff
@@ -0,0 +1,111 @@
...
+import { initReactI18next } from 'react-i18next';
+
+i18n
+  .use(LanguageDetector)
+  .use(initReactI18next)
+  .init({
+    debug: true,
+    fallbackLng: 'en',
+    resources: {
+      en: {
```
Comment (mazhula97) [inline src/i18n.ts:14]: Move it to the separate files(src/locales). For example: en.json, ua.json

File: src/locales/en.json [hunk 2]
```diff
@@ -0,0 +1,69 @@
+{
+    "translation": {
```
Comment (mazhula97) [inline src/locales/en.json]: Don't need `translation` key

File: src/i18n.ts [hunk 3]
```diff
@@ -0,0 +1,23 @@
...
+        translation: en.translation,
```
Comment (mazhula97) [inline src/i18n.ts]: Don't need `translation` key

File: src/components/form/signIn/index.tsx [hunk 4]
```diff
@@ -13,6 +14,8 @@ import {
 
 const SignInForm = () => {
   const { handleSubmit, register, errors, onSignInSubmit } = useAuthForm();
+  const { t } = useTranslation();
```
Comment (mazhula97) [inline src/components/form/signIn/index.tsx:17]: In general you can return `t` from `useFormHelpers` and don't use an additional `useTranslation()`

**Review by mazhula97 — CHANGES_REQUESTED** [review]

File: src/i18n.ts (hunk 1 above)
Comment (VladyslavGilyuk) [reply src/i18n.ts:14]: Fix: moved to separate files

File: src/locales/en.json (hunk 2 above)
Comment (VladyslavGilyuk) [reply src/locales/en.json]: fix: translation key removed in both en & uk files

File: src/i18n.ts (hunk 3 above)
Comment (VladyslavGilyuk) [reply src/i18n.ts]: fix: translation key removed both for en & uk versions

File: src/components/form/signIn/index.tsx (hunk 4 above)
Comment (VladyslavGilyuk) [reply src/components/form/signIn/index.tsx:17]: fix: useTranslation removed. Now useFormHelpers accepts 't' as a prop =>
const useFormHelpers   = (t: (key: string) => string) 
Exp: const { SingUpFormHelper } = useFormHelpers(t);

Comment (mazhula97) [reply src/components/form/signIn/index.tsx:17]: You could make a vice versa. const { t, SignInFormHelper } = useFormHelpers;
But ok, thats fine

**Review by mazhula97 — APPROVED** [review]

**Pull request merged**